og/bump/action.yml

```diff
@@ -0,0 +1,33 @@
+name: bump
+description: |
+  Automatically bump the version file of the OG repo
+
+inputs:
+  base_branch:
+    description: The base branch of the PR.
+    required: true
+  pr_body:
+    description: OG formatted body of the PR description.
+    required: true
+  pr_title:
+    description: SC formatted title of the PR.
+    required: true
+  dev_branch:
+    description: The dev branch of this action to connect to the workflow
+    required: false
+    default: 'main'
```

Comment:
Oof. Maybe we should switch from `master` to `main` everywhere in Q4?

Reply:
`main` is this repo's default branch.

Reply:
Ah, ok, this is just for _this_ repo, and not others making use of it.

Reply:
I recently tried switching from `master` to `main` in one of my private repos, it went smoother than I expected. The only break points I'd expect here are really external references to the branch name, and even there I'm not sure if github doesn't autocorrect checking out `master` then. I think we should just do it in Q4.